nav2_planner/src/planner_server.cpp

```diff
@@ -461,26 +461,52 @@ PlannerServer::computePlan()
       return;
     }
 
-    result->path = getPlan(start, goal_pose, goal->planner_id);
+    try
+    {
+      result->path = getPlan(start, goal_pose, goal->planner_id);
 
-    if (!validatePath(action_server_pose_, goal_pose, result->path, goal->planner_id)) {
-      return;
-    }
+      if (!validatePath(action_server_pose_, goal_pose, result->path, goal->planner_id)) {
+        return;
+      }
 
-    // Publish the plan for visualization purposes
-    publishPlan(result->path);
+      // Publish the plan for visualization purposes
+      publishPlan(result->path);
 
-    auto cycle_duration = steady_clock_.now() - start_time;
-    result->planning_time = cycle_duration;
+      auto cycle_duration = steady_clock_.now() - start_time;
+      result->planning_time = cycle_duration;
 
-    if (max_planner_duration_ && cycle_duration.seconds() > max_planner_duration_) {
-      RCLCPP_WARN(
-        get_logger(),
-        "Planner loop missed its desired rate of %.4f Hz. Current loop rate is %.4f Hz",
-        1 / max_planner_duration_, 1 / cycle_duration.seconds());
-    }
+      if (max_planner_duration_ && cycle_duration.seconds() > max_planner_duration_) {
+        RCLCPP_WARN(
+            get_logger(),
+            "Planner loop missed its desired rate of %.4f Hz. Current loop rate is %.4f Hz",
+            1 / max_planner_duration_, 1 / cycle_duration.seconds());
+      }
 
-    action_server_pose_->succeeded_current(result);
+      action_server_pose_->succeeded_current(result);
+    }
+    catch (nav2_core::GlobalPlannerStartOccupiedException &e)
+    {
```

Comment:
Some logging for each would be nice (like the default exception catching already in the server)